keras_cv/layers/preprocessing/resizing.py

```diff
@@ -273,22 +273,6 @@ def resize_with_crop_to_aspect(x):
         inputs["images"] = images
         return inputs
 
-    def call(self, inputs, training=True):
```

Comment:
I don't see any references to `training` here!  What behavior are we removing here?